package.json

```diff
@@ -28,7 +28,7 @@
     "clean:windows": "node -e \"if (process.platform === 'win32') process.exit(1)\" || , if exist dist rmdir /Q /S dist",
     "devnet:start": "npx --yes @alephium/cli@latest devnet start",
     "devnet:stop": "npx --yes @alephium/cli@latest devnet stop",
-    "compile": "npx cli compile",
+    "compile": "npx cli compile && npx ts-node stats",
```

Comment:
imo it will be better to run "npx cli compile && npm run stats", so we maintain single source of running the script